ktlint-cli/src/main/scripts/ktlint.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
+setlocal
```

Comment:
`set` command is equivalent to `export` in Unix, meaning it'll change the surrounding environment. Executing `setlocal` will ensure that `JAR_PATH` is only a local variable and it only affects the single command after it's set.